src/util/socket-utils.c

```diff
@@ -881,9 +906,15 @@ open_super_socket(const char* socket_name)
 		for (; NULL != options; options = strchr(options+1, ',')) {
 			if (strcasehasprefix(options, ",b") && isdigit(options[2])) {
 				// Change Baud rate
-				int baud = baud_rate_to_termios_constant((int)strtol(options+2,NULL,10));
+				const int baud = (int)strtol(options+2,NULL,10);
+				const int baud_opt = baud_rate_to_termios_constant(baud);
+				if(baud_opt <= 0) {
+					syslog(LOG_ERR, "Unsupported baud rate %d", baud);
```

Comment:
Thoughts on closing the file descriptor here and returning -1 instead of continuing?

Reply:
I agree. Returning -1 sound good here.

Reply:
Updated PR